### PR TITLE
[fix](stats) fix auto collector always create sample job no matter the table size

### PR DESCRIPTION
**Pull request by Kikyou1997**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public class StatisticsAutoCollector extends StatisticsCollector {
 
     public StatisticsAutoCollector() {
         super("Automatic Analyzer",
-                TimeUnit.MINUTES.toMillis(Config.auto_check_statistics_in_minutes),
+                TimeUnit.SECONDS.toMillis(Config.auto_check_statistics_in_minutes),
```
Comment (Jibing-Li) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java:53]: Should this config item auto_check_statistics_in_minutes changed to auto_check_statistics_in_seconds?

File: fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisInfo.java [hunk 2]
```diff
@@ -182,14 +182,19 @@ public enum ScheduleType {
     @SerializedName("usingSqlForPartitionColumn")
     public final boolean usingSqlForPartitionColumn;
 
+    /**
+     * Used to store the newest partition version of tbl when creating this job.
+     */
+    public final long tblUpdateTime;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisInfo.java:188]: do not need presist?

File: fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java (hunk 1 above)
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java:53]: why change to second?

**Comment by Kikyou1997** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisInfo.java (hunk 2 above)
Comment (Kikyou1997) [reply fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisInfo.java:188]: No, this info would finally save in TableStatsMeta

**Review by Jibing-Li — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisInfo.java (hunk 2 above)
Comment (morrySnow) [reply fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisInfo.java:188]: add comment to this variable, it is very confuse

**Comment by Kikyou1997** [comment]
run  buildall

**Comment by morrySnow** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

**Comment by Kikyou1997** [comment]
run buildall

**Review by englefly — DISMISSED** [review]

**Comment by Jibing-Li** [comment]
run buildall

**Comment by hello-stephen** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Pull request merged**